banana/templates/banana/image_detail.html

```diff
@@ -45,18 +44,24 @@
 
                     <dt>Filename</dt>
                     <dd>{{ object.filename }}</dd>
+                    
+                    <dt>Rejections</dt>
+                    <dd>{{ object.rejections.all|yesno }} </dd>
 
 
                 </dl>
             </div>
             <div class="col-md-4">
                 <dl class="dl-horizontal">
-                    <dt>Rejections</dt>
-                    <dd>{{ object.rejections.all|yesno }} </dd>
 
-                    <dt>Number of sources</dt>
+                    <dt># of sources</dt>
                     <dd>{{ object.extractedsources.count }}</dd>
 
+                    <dt># of blind extractions</dt>
+                    <dd>{{ object.blind_extractedsources.count }}</dd>
+
+                    <dt># of forced extractions</dt>
```

Comment:
Yeah I was worried about that...

What about just 'blind sources' and 'forced sources', does it really need to be explicit in saying 'number of'?